pymongo/mongo_client.py

```diff
@@ -109,11 +109,9 @@ def __init__(
 
         The `host` parameter can be a full `mongodb URI
         <http://dochub.mongodb.org/core/connections>`_, in addition to
-        a simple hostname. It can also be a list of hostnames or
-        URIs. Any port specified in the host string(s) will override
-        the `port` parameter. If multiple mongodb URIs containing
-        database or auth information are passed, the last database,
-        username, and password present will be used.  For username and
+        a simple hostname. It can also be a list of hostnames but no more
+        than one URI. Any port specified in the host string(s) will override
+        the `port` parameter. For username and
```

Comment:
We need to make a similar change to the description in the `:Parameters:` list below.

Reply:
Done.